### PR TITLE
6.0.19 updates

### PR DESCRIPTION
**Pull request by Amerlander**
- update assets
- update user settings
- copy approved repos to `approvedRepoLib`

### CODE REVIEW

**Comment by Amerlander** [comment]
@abchatra can you merge and bump?

File: libs/core-mini/pxt.json [hunk 1]
```diff
@@ -11,5 +11,42 @@
...
+                        "bluetooth": {
+                            "open": 1,
+                            "security_level": null,
+                            "whitelist": 0
+                        }
+                    }
+                }
+            },
+            {
+                "description": "Bluetooth: Paring required (Android 13 »GATT ERROR 133«-Bug)",
```
Comment (Amerlander) [inline libs/core-mini/pxt.json:39]: @joernalraun

**Comment by joernalraun** [comment]
> @abchatra can you merge and bump?

... or maybe @riknoll?

**Review by abchatra — APPROVED** [review]

**Comment by abchatra** [comment]
Done

**Pull request merged**